src/app/containers/Image/index.stories.jsx

```diff
@@ -38,9 +38,10 @@ const dataWithCaption = imageData([
   ),
 ]);
 
-storiesOf('Image', module).add('image with alt text', () => (
-  <Image {...dataWithAltText} />
-));
-storiesOf('Image', module).add('image with alt text and caption', () => (
-  <Image {...dataWithCaption} />
+storiesOf('ImageContainer', module).add('image with alt text', () => (
+  <ImageContainer {...dataWithAltText} />
```

Comment:
can we just call this data or dataWithoutCaption the inconsistent naming between this and below (which doesn't specify that it has alt text is confusing.

Reply:
same in the test file